### ISSUE TITLE
Game crashes when trying to get the "Book of Secrets" or the "Book of Face" goodies

### ISSUE THREAD

**Issue by muntdefems** [larsiusprime/tdrpg-bugs#1035]
# Basic Info
- Arch Linux 64bit (kernel version 4.6.2-1)
- Download
- DQ1 DX Version (from GOG.com)
- Version # 1.0.1
- Graphics card: NVIDIA GeForce GTX 650 with proprietary drivers (v364.19)
- PC: Desktop with a monitor capable of a 1920x1080 resolution

# Steps to Reproduce:
1. Launch the game
2. Click the "Bonus Goodies" in the main menu screen
3. Click the blue "Okay!" button

# What I expected to happen:
I expected a browser window to open and to get the goodies in-game.

# What happened instead:
The game crashed while outputting the following message:
```
xdg-open: unexpected argument '&'
Try 'xdg-open --help' for more information.
```

**Comment by larsiusprime** [comment]
Thanks, we'll try to get this sorted in the next 24 hours.

On Tue, Jun 14, 2016 at 4:52 PM, muntdefems <notifications@github.com>
wrote:

> Basic Info
>
>    - Arch Linux 64bit (kernel version 4.6.2-1)
>    - Download
>    - DQ1 DX Version (from GOG.com)
>    - Version # 1.0.1
>    - Graphics card: NVIDIA GeForce GTX 650 with proprietary drivers
>    (v364.19)
>    - PC: Desktop with a monitor capable of a 1920x1080 resolution
>
> Steps to Reproduce:
>
>    1. Launch the game
>    2. Click the "Bonus Goodies" in the main menu screen
>    3. Click the blue "Okay!" button
>
> What I expected to happen:
>
> I expected a browser window to open and to get the goodies in-game.
> What happened instead:
>
> The game crashed while outputting the following message:
>
> xdg-open: unexpected argument '&'
> Try 'xdg-open --help' for more information.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/larsiusprime/tdrpg-bugs/issues/1035>, or mute the
> thread
> <https://github.com/notifications/unsubscribe/AAtG-A5nvB1vEfRYTc3wpg_diF7xY7iBks5qLyKdgaJpZM4I1zLf>
> .
>



-- 
www.fortressofdoors.com -- Games, Art, Design

**Comment by muntdefems** [comment]
No, thank you! :)

PS: I couldn't figure out how to add labels to the issue, sorry...

**Comment by ManDude** [comment]
Only collaborators can. But don't worry about that. :)

**Comment by Skiski** [comment]
I have the same problem. I thought it was linked to the infamous close/close/close bug, but now it is gone, I have this problem. But I managed to get the book of face.

**Comment by larsiusprime** [comment]
You can work around it by saying "no" to the narrator and refusing to click yes. Turns out the issue is caused by calling a utility to open your web browser...

**Comment by muntdefems** [comment]
The issue seems to be fixed with the new "DQ1 2.0.2a" version from GOG.com.

**Comment by larsiusprime** [comment]
Great, thanks!
